src/types.ts

```diff
@@ -18,7 +18,7 @@ export type ThemeOptions = {
   /**
    * @default 'zinc'
    */
-  color?: ColorOptions
+  color?: ColorOptions | null
```

Comment:
Maybe use `false`?

Reply:
Sure, I think I'll also add false for radius?

Reply:
Yes, It's OK